src/Features/Core/Portable/Diagnostics/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -17,8 +17,9 @@ internal static class IDEDiagnosticIds
         public const string AddBracesDiagnosticId = "IDE0011";
         public const string PreferIntrinsicPredefinedTypeInDeclarationsDiagnosticId = "IDE0012";
         public const string PreferIntrinsicPredefinedTypeInMemberAccessDiagnosticId = "IDE0013";
-        public const string PreferFrameworkTypeInDeclarationsDiagnosticId = "IDE0014";
-        public const string PreferFrameworkTypeInMemberAccessDiagnosticId = "IDE0015";
+        public const string PreferFrameworkTypeDiagnosticId = "IDE0014";
+        // Not used any more.  All 'prefer framework type' diagnostics go through the above ID
+        // public const string PreferFrameworkTypeInMemberAccessDiagnosticId = "IDE0015";
```

Comment:
❗️ This is a behavior change, not just a code simplification. The pull request initial comment needs to be clear in cases like this so design changes aren't accidentally overlooked thinking it was a behavior-preserving refactoring.

Reply:
Sorry, i could have sworn i actually added a PR comment to that effect.  I must not have submitted it.  That was bad on my part and was **not**  intentional.

Reply:
This is very weird.  I def wrote up something for this, but i cannot find it.  It specifically called out the change here and how this could affect the user if they had a specific suppression.  However, i also called out that this was the behavior of many of our features (like 'use var').  We don't have an ID for 'when apperant' vs 'when built-in' vs 'always'.  Instead, we just allow people have specify the option for 'use var' and then we use a single ID for all cases.